### PR TITLE
BETA: Register afrtite.is-a.dev

### PR DESCRIPTION
**Pull request by 5enox**
Added `afrtite.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by 5enox** [comment]
guys i really need the domain name please

File: domains/abdo.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "5enox",
+        "email": "animikantan@gmail.com"
+    },
+    "record": {
+        "CNAME": "my-proto.onrender.com"
```
Comment (DEV-DIBSTER) [inline domains/abdo.json:7]: Unable to access CNAME.

File: domains/afrtite.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "5enox",
+        "email": "animikantan@gmail.com"
+    },
+    "record": {
+        "CNAME": "my-proto.onrender.com"
```
Comment (DEV-DIBSTER) [inline domains/afrtite.json:7]: Unable to access CNAME.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/afrtite.json (hunk 2 above)
Comment (5enox) [reply domains/afrtite.json:7]: fixed it just now

Comment (5enox) [reply domains/afrtite.json:7]: I'm using render.com to host my flask site for now because it's free

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**